spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -143,7 +141,7 @@
       when_I_submit_the_form
       then_I_should_see "has been scheduled to publish"
       and_the_step_by_step_should_have_the_status "Scheduled"
-      and_there_should_be_a_change_note "Minor update scheduled by Test author for publishing at #{schedule_time}"
+      and_there_should_be_a_change_note "Minor update scheduled by Test author for publishing at 2030-04-20 10:26:00 +0100"
```

Comment:
Could you fix this up to the first commit?

Reply:
I could do, but the `schedule_time` variable is used in two places at that point, so it's nice and DRY.

In this commit, we're no longer using `schedule_time` in `and_I_fill_in_the_scheduling_form`, as the inputs have been split up, so it no longer makes sense to have a variable, hence the hard-coding here.